dpe/Cargo.toml

```diff
@@ -9,6 +9,17 @@ edition = "2021"
 default = ["dpe_profile_p256_sha256"]
 dpe_profile_p256_sha256 = ["platform/dpe_profile_p256_sha256"]
 dpe_profile_p384_sha384 = ["platform/dpe_profile_p384_sha384"]
+arbitrary_max_handles = []
```

Comment:
nit: can you add a comment here explaining how to use this? (Basically just what you put in the PR description)

Reply:
Done